map/track.cpp

```diff
@@ -46,15 +46,17 @@ std::vector<double> Track::GetLengthsImpl() const
   auto const & lines = GetData().m_geometry.m_lines;
   std::vector<double> lengths;
   lengths.emplace_back(0.0);
```

Comment:
Remove this emplcase_back.

Postcondition: lengths.size() == SUM(lines[i].size())